Neos.Fusion/Classes/FusionObjects/DataStructureImplementation.php

```diff
@@ -96,6 +98,21 @@ private function isUntypedProperty($property): bool
         if (!is_array($property)) {
```

Comment:
by making `private function isUntypedProperty(`

protected - we could override it in subclasses eg. in Join and always return false there ...